### PR TITLE
Fix various issues encountered during release process

### PR DESCRIPTION
**Pull request by Kami**
We recently converted all of our Mistral workflows to Orquesta.

We are now running first release with those workflows and various issues are popping up.

This pull request includes fixes for those various issues.

### CODE REVIEW

File: actions/workflows/mistral_get_build_hosts.yaml [hunk 1]
```diff
@@ -29,5 +29,5 @@ tasks:
     next:
       - when: <% succeeded() %>
         publish:
-          - hosts: <% dict(test => result().host,
-                           build => result().host) %>
+          - hosts: <% dict(test => task(get_test_host).result.output.host,
+                           build => test(get_build_host).result.output.host) %>
```
Comment (armab) [inline actions/workflows/mistral_get_build_hosts.yaml]: you probably want `task()` instead of `test()` here?

Comment (armab) [inline actions/workflows/mistral_get_build_hosts.yaml]:
```suggestion
                           build => task(get_build_host).result.output.host) %>
```

Comment (Kami) [reply actions/workflows/mistral_get_build_hosts.yaml]: Yep, a typo :)

**Comment by blag** [comment]
I think this task has been successfully publishing into the parent workflow. See execution 5c6afa3e16f3ad3417fcbd3f.

Task output appears fine to me:

```json
{
  "output": {
    "hosts": {
      "test": "10.x.y.z",
      "build": "10.x.y.z"
    }
  }
}
```

**Comment by Kami** [comment]
Closing - not relevant anymore.

**Pull request closed without merging**